freesound/settings.py

```diff
@@ -199,6 +199,9 @@
 # Number of ratings of a sound to start showing average
 MIN_NUMBER_RATINGS = 3
 
+# Buffer size for CRC computation
+CRC_BUFFER_SIZE = 4096
```

Comment:
Does this really need to be a settings.py wide setting? This seems a bit weird.

Reply:
I agree that maybe it is too global to put in into the settings.py, but don't we want to have it configurable? If not, is it ok to have 'magic' numbers in the code? I can always leave it as it was in your code example

Reply:
I think it's fine to have it in `settings.py`. We'll have organize it a bit a some point but I'd argue that's fine.